doc/source/whatsnew/v2.1.0.rst

```diff
@@ -42,10 +42,9 @@ Other enhancements
 - Improved error message when creating a DataFrame with empty data (0 rows), no index and an incorrect number of columns. (:issue:`52084`)
 - :meth:`DataFrame.applymap` now uses the :meth:`~api.extensions.ExtensionArray.map` method of underlying :class:`api.extensions.ExtensionArray` instances (:issue:`52219`)
 - :meth:`arrays.SparseArray.map` now supports ``na_action`` (:issue:`52096`).
```

Comment:
Can you remove the period at the end of this line? I believe is it messing up our pre-commit script.

Reply:
cc @MarcoGorelli - just pinging for awareness.

Reply:
@rhshadrach Just removed the period and pushed.